build.sbt

```
@@ -33,12 +33,32 @@ developers += Developer(
 organizationName := "Lightbend Inc."
 startYear        := Some(2018)
 
-bintrayOrganization := Some("sbt")
-bintrayRepository   := "sbt-plugin-releases"
-
 enablePlugins(AutomateHeaderPlugin)
 scalafmtOnCompile := true
 
-// don't do any API docs
-doc / sources                := Seq()
-packageDoc / publishArtifact := false
+ThisBuild / githubWorkflowTargetTags ++= Seq("v*")
+ThisBuild / githubWorkflowPublishTargetBranches :=
+  Seq(RefPredicate.StartsWith(Ref.Tag("v")))
+ThisBuild / githubWorkflowPublish := Seq(
+  WorkflowStep.Sbt(
+    List("ci-release"),
+    env = Map(
+      "PGP_PASSPHRASE" -> "${{ secrets.PGP_PASSPHRASE }}",
+      "PGP_SECRET" -> "${{ secrets.PGP_SECRET }}",
+      "SONATYPE_PASSWORD" -> "${{ secrets.SONATYPE_PASSWORD }}",
+      "SONATYPE_USERNAME" -> "${{ secrets.SONATYPE_USERNAME }}"
+    )
+  )
+)
+
+ThisBuild / test / publishArtifact := false
+ThisBuild / pomIncludeRepository   := (_ => false)
+sonatypeProfileName                := "com.lightbend"
+
+ThisBuild / githubWorkflowJavaVersions := List(
+  JavaSpec.temurin("8"),
+  JavaSpec.temurin("11"),
+  JavaSpec.temurin("17")
+)
+
+ThisBuild / githubWorkflowTargetBranches := Seq("master")
```

maybe already

```suggestion
ThisBuild / githubWorkflowTargetBranches := Seq("main", "master")
```

for if/when we move to that - not important though

---

I can do this but its different to the other lightbend projects where I integrated sbt-paradox-dependencies so it would be inconsistent in this regard. Would you be happy if I made an issue stating that you also need to do `ThisBuild / githubWorkflowTargetBranches` when changing from `main` to `master`?

Also I think that @ennru is familiar with sbt-github-actions and is aware of this.

---

> I can do this but its different to the other lightbend projects where I integrated sbt-paradox-dependencies so it would be inconsistent in this regard.

Ok

> Would you be happy if I made an issue stating that you also need to do `ThisBuild / githubWorkflowTargetBranches` when changing from `main` to `master`?

I don't think an issue is strictly necessary, we'll notice soon enough when we make the switch :)

---

Let's add `main` so that we can do the branch rename.